SkeletalRepresentationInitializer/Logic/vtkSlicerSkeletalRepresentationInitializerLogic.cxx

```diff
@@ -1509,6 +1509,7 @@ int vtkSlicerSkeletalRepresentationInitializerLogic::ComputePairwiseTps(int tota
         curveFileName = tempFolder + "/model/curve" + std::to_string(stepNum-1) + ".vtk"; // output
         TransformPoints(tps, curvePoly, curveFileName);
     }
+    return 0;
```

Comment:
If the return is always 0, make the function `void` instead of `int` and remove the return.